examples/fluid/sphere_surface_tension_2d.jl

```diff
@@ -1,28 +1,28 @@
 # In this example we can observe that the `SurfaceTensionAkinci` surface tension model correctly leads to a
-# surface minimization of the water square and approaches a circle.
+# surface minimization of the water square and approaches a sphere.
 using TrixiParticles
 using OrdinaryDiffEq
 
 fluid_density = 1000.0
 
 particle_spacing = 0.1
+square_size = 0.5
```

Comment:
```suggestion
fluid_size = (0.5, 0.5)
```
More consistent with the other examples. Less confusing for users to work with `trixi_include`. Especially when using the 3D example as well.